python_modules/dagster-graphql/dagster_graphql/implementation/fetch_assets.py

```diff
@@ -75,6 +78,34 @@ def _normalize_asset_cursor_str(cursor_string: Optional[str]) -> Optional[str]:
         return cursor_string
 
 
+def get_asset_records(
+    graphene_info: "ResolveInfo",
+    prefix: Optional[Sequence[str]] = None,
+    cursor: Optional[str] = None,
+    limit: Optional[int] = None,
+) -> "GrapheneAssetRecordConnection":
+    from dagster_graphql.schema.pipelines.pipeline import GrapheneAssetRecord
+    from dagster_graphql.schema.roots.assets import GrapheneAssetRecordConnection
+
+    instance = graphene_info.context.instance
+
+    normalized_cursor_str = _normalize_asset_cursor_str(cursor)
+    materialized_assets = sorted(
+        instance.get_asset_keys(prefix=prefix, limit=limit, cursor=normalized_cursor_str), key=str
+    )
```

Comment:
if were punting on it leave a comment that the expectation is that this will get replaced with `get_asset_records` once the pagination arguments are exposed